launch-environment

```diff
@@ -4,7 +4,4 @@ AWS_SECRET_ACCESS_KEY
 
 # AWS Configuration
 AWS_REGION
-AWS_DEFAULT_REGION
```

Comment:
Why do these go? The golang AWS library is difficult and non-obvious WRT to these, I can't recall the detail as to why they're all there but they were necessary once